### ISSUE TITLE
Duplicate pivot actions

### ISSUE THREAD

**Issue by zaratedev** [laravel/nova-issues#3344]
- Laravel Version: 8.35.1
- Nova Version: 3.22.0
- PHP Version: 7.4.16
- Database Driver & Version: Maria db 10.5.9

### Description:
When we define a pivot action in a `BelongsToMany` field in the resource detail, the pivot action is shown twice.

### Detailed steps to reproduce the issue on a fresh Nova installation:

Define belongs to many relationship on the user resource.

```php
BelongsToMany::make('Contests')->actions(fn () => [new ChooseWinner]);
```

![Screen Shot 2021-04-14 at 17 33 03](https://user-images.githubusercontent.com/29809845/114790582-9c476f00-9d4a-11eb-8f12-a2972771cec7.png)

**Comment by zaratedev** [comment]
@davidhemphill  Any solution to this problem?

**Comment by crynobone** [comment]
@zaratedev PR has been merged but have to wait for next patch release.

**Comment by zaratedev** [comment]
Great!

**Comment by crynobone** [comment]
Fixed in [v3.24.0 release](https://nova.laravel.com/releases/3.24.0).
